config/params.php

```diff
@@ -72,10 +73,13 @@
         'schema-providers' => [],
 
         /**
-         * Config for {@see \Yiisoft\Yii\Cycle\Schema\Conveyor\AnnotatedSchemaConveyor}
-         * Annotated entity directories list.
+         * Annotated/attributed entity directories list.
          * {@see \Yiisoft\Aliases\Aliases} are also supported.
          */
+        'entity-paths' => [],
+        'conveyor-class' => CompositedSchemaConveyor::class,
+
+        /** @deprecated use `entity-paths` key instead */
```

Comment:
No need `deprecated` since package isn't stable.

Reply:
in a weak or later we can remove this option. But now this deprecation won't break existing configs